pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.1 Release Notes
+title: Kommander 2.1.1 Release Notes
+menuWeight: 20
+excerpt: View release-specific information for Kommander 2.1.1
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.0 was released on December 23, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## New features and capabilities
+
+### Ability to add catalog applications 
+
+With this release, you can now add catalog applications to your Kommander instances. Currently, we provide the following operator applications that are workspace-scoped:
+
+* Kafka
+* Spark
+* Zookeeper
+
+For more information, see the catalog application documentation. 
+
+## Fixes and Improvements
+
+- Fixed an issue where the 'dkp delete' command could fail with a SIGSEGV when attempting to delete a DKP cluster from AWS where you have permanent credentials. (COPS-7109)
+- Fixed an issue where the AWS `--region` or Azure `--location` installer flags were not being enforced in the target cluster. (COPS-7101)
+- Corrected an issue where the `PreprovisionedInventory` object and SSH key secret were not moved to the target cluster when making the cluster self-managing.(COPS-7079)
+
+## Component updates
+
+The following services and service components have been upgraded to the listed version:
+
+- centralized-grafana: 18.1.1
+- centralized-kubecost: 0.20.0
+- cert-manager: 0.2.7
+- dex: 2.20.5
```

Comment:
I see `2.9.10` for `dex` 🤔, maybe this is was meant to be the `external-dns` version below which is `2.20.5`
```suggestion
- dex: 2.9.10
```

Reply:
hmm maybe we should list both?

Reply:
yeah maybe someone overly deleted (could have been me idk)